recipes/compute.rb

```diff
@@ -47,23 +49,23 @@
     end
 
     # Install latest version included in the repo above
-    package "kernel" do
+    package 'kernel' do
       version node['osl-openstack']['openpower']['kernel_version']
       action :upgrade
     end
-  end
-
-  # Turn off smt on boot (required for KVM support)
-  # NOTE: This really should be handled via an rclocal cookbook
-  cookbook_file "/etc/rc.d/rc.local" do
-    owner "root"
-    group "root"
-    mode 0755
-  end
 
-  # Turn off smt during runtime
-  execute "ppc64_cpu_smt_off" do
-    command "/sbin/ppc64_cpu --smt=off"
-    not_if "/sbin/ppc64_cpu --smt | grep 'SMT is off'"
+    # Turn off smt on boot (required for KVM support)
+    # NOTE: This really should be handled via an rclocal cookbook
+    cookbook_file '/etc/rc.d/rc.local' do
+      owner 'root'
+      group 'root'
+      mode 0755
+    end
+    
+    # Turn off smt during runtime
+    execute 'ppc64_cpu_smt_off' do
+      command '/sbin/ppc64_cpu --smt=off'
+      not_if '/sbin/ppc64_cpu --smt | grep \'SMT is off\''
+    end
```

Comment:
While this will likely never be run on fedora on non ppc64 machines, I extended the case statement to here because:

1) It exists, so it might as well cover everything.
2) It will make it so that if someone runs these tests on an x86 fedora vm by accident won't fail with unexpected errors